src/main/java/uk/gov/pay/directdebit/gatewayaccounts/api/GatewayAccountResponse.java

```diff
@@ -33,36 +34,35 @@
     private String type;
     @JsonProperty("analytics_id")
     private String analyticsId;
-    @JsonProperty("is_connected")
-    private final boolean isConnected;
+    @JsonProperty("organisation")
```

Comment:
We should have defined what to call it in the story. I think `provider_id` would be consistent with payments and mandates — “organisation” is kind of GoCardless-specific.